Shuttle-back/src/main/resources/data.sql

```diff
@@ -38,13 +38,23 @@ insert into vehicle(vehicle_type_id, driver_id) values(1, 1);
 
 insert into location(address, latitude, longitude) values ('Street 1', 45.267136, 19.833549);
 insert into location(address, latitude, longitude) values ('Street 2', 44.267136, 20.833549);
+insert into location(address, latitude, longitude) values ('Street 3', 44.267136, 19.833549);
+insert into location(address, latitude, longitude) values ('Street 4', 45.267136, 20.833549);
 
+insert into route() values();
 insert into route() values();
 
 insert into route_locations(route_id, locations_id) values(1, 1);
 insert into route_locations(route_id, locations_id) values(1, 2);
 
-insert into ride(status, driver_id, route_id, vehicle_type_id, pet_transport, baby_transport) values (0, 1, 1, 1, true, false);
+insert into route_locations(route_id, locations_id) values(2, 3);
+insert into route_locations(route_id, locations_id) values(2, 4);
+
+insert into ride(status, driver_id, route_id, vehicle_type_id, pet_transport, baby_transport, end_time) values (0, 1, 1, 1, true, false,CURRENT_TIMESTAMP);
```

Comment:
Ride status 0 is 'Pending'. Why is `end_time` being set to `CURRENT_TIMESTAMP` while `start_time` is not initialized at all? What does this achieve?